src/pages/Pool/index.tsx

```diff
@@ -225,7 +226,6 @@ export default function Pool() {
   }
 
   const showConnectAWallet = Boolean(!account)
-  const showV2Features = Boolean(V2_FACTORY_ADDRESSES[chainId])
```

Comment:
My only issue here is that you're adding a new file to the V2 checking codepath, which complicates adding add'l chains. However, I don't think we plan on adding add'l chains to V2, so this seems fine.